src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/matcher/DefendantMatcher.kt

```diff
@@ -11,7 +11,7 @@ class DefendantMatcher(defendants: List<DefendantEntity>?, person: Person) :
     return person.otherIdentifiers?.pncIdentifier == PNCIdentifier.from(item.pncNumber?.pncId) ||
       item.firstName.equals(person.givenName, true) ||
       item.surname.equals(person.familyName, true) ||
-      item.dateOfBirth == person.dateOfBirth
+      (item.dateOfBirth != null && item.dateOfBirth == person.dateOfBirth)
```

Comment:
This stops null dates from matching in the first place if it is the only match. Definitely a bodge - we need to do a lot better in here. 
Far too many matches

Reply:
I think this will be gone next week. We don't need any of the matchers.